### ISSUE TITLE
start a new design

### ISSUE THREAD

**Issue by lageist** [UNC-Project-Open-AAC/OS-DPI#160]
The start a new design link [here](https://github.com/UNC-Project-Open-AAC/OS-DPI) opens a project that already has components in the layout.

**Comment by gbishop** [comment]
Yes, it simply opens a design named "new". If you already have one, it will
still be there.

**Comment by gbishop** [comment]
This feels like a bug... The New entry in the File menu does the right thing. 

I've wondered if we should remove these links from the README and put some similar ones on the Welcome page.
